src/cls/_ZPM/PackageManager/Server/SemVerIndex.cls

```diff
@@ -4,7 +4,7 @@
 /// <li>Simplify ordering based on latest semantic version, properly handling major/minor/patch version numbers and prerelease designations.</li>
 /// </ol>
 /// </p>
```

Comment:
Perhaps rename to %ZPM.General.SemanticVersion.Index? I added it to the google sheet